api/charity.py

```diff
@@ -20,4 +32,8 @@ def __init__(self):
         nickname='Get Supported Charities')
     def get(self):
         """Get Supported Charities"""
+        # Format the JSON response
```

Comment:
these lines aren't needed. the front-end can handle the normal `return SUPPORTED_CHARITIES, 200`